src/webui/frontend/src/actions/auth.ts

```diff
@@ -27,17 +27,19 @@ export const LoginActions = {
 
 export type LoginActions = ActionsUnion<typeof LoginActions>;
 
-export const oauthRedirect = (account: string, role: string, database: string, warehouse: string, returnHref: string) => async (dispatch: Dispatch) => {
-  try {
-    const response = await api.oauthRedirect({account, role, database, warehouse, returnHref});
-    if (response.url) {
-      console.log('navigating', response.url);
-      navigate(response.url);
+export const oauthRedirect =
```

Comment:
just reformatting, no change to code here, right?

Reply:
Yes, no code changes here.